src/main/java/com/palechip/hudpixelmod/HudPixelConfig.java

```diff
@@ -73,15 +75,15 @@
     public static final String PAINTBALL_KILLSTREAK_TRACKER = "Show/Hide your active Paintball killstreaks and the ones on cooldown.";
     public static final String KILLSTREAK_TRACKER = "Enable/Disable tracking how many kills you can get within one live in ";
     public static final String BLITZ_DEATHMATCH_NOTIFIER = "Show/Hide a little flashing message when death match is near in Blitz Survival Games";
-    public static final String WITHER_COIN_COUNTER  = "Turn on/off the Counting of the Coins for Wither Damage in Mega Walls.";
+    public static final String WITHER_COIN_COUNTER = "Turn on/off the Counting of the Coins for Wither Damage in Mega Walls.";
 
     public HudPixelConfig(File config) {
         // the "3" is the defined config version
-        this.config = new Configuration(config, "3");
+        this.config = new Configuration(config, "4");
```

Comment:
Why? This is not necessary for adding new settings. It is only needed for renaming and updating existing settings.